react/react-basics/step1.md

```diff
@@ -1,6 +1,6 @@
 A card is a flexible element for containing any type of content. Cards are used on dashboards, in data displays, or for positioning content on a page.
 
-In PatternFly, separate components are used to demarcate the different regions of a card so it can be fully customized. For example, images are placed in `CardHead`, actions in `CardActions`, and the header in `CardHeader`.
+In PatternFly, separate components are used to demarcate the different regions of a card so it can be fully customized. For example, images are placed in `CardHeaderMain`, actions in `CardActions`, and the header in `CardTitle`.

```

Comment:
wdyt about using a more intelligible word here? Also, we have a `<CardHeader>` - is it confusing to say the "header" goes in `<CardTitle>`? I wonder if we should update our component examples, too, to change the text "header" to "title".

```suggestion
In PatternFly, separate components are used to define the different regions of a card so it can be fully customized. For example, images are placed in `CardHeaderMain`, actions in `CardActions`, and the title in `CardTitle`.

```